src/ServiceControl.Audit/Infrastructure/SetupBootstrapper.cs

```diff
@@ -102,6 +103,17 @@ public async Task Run(string username)
 
                 await Endpoint.Create(configuration)
                     .ConfigureAwait(false);
+
+
+                var ravenOptions = new RavenStartup();
+                foreach (var indexAssembly in RavenBootstrapper.IndexAssemblies)
+                {
+                    ravenOptions.AddIndexAssembly(indexAssembly);
+                }
+
+                var service = new EmbeddedRavenDbHostedService(documentStore, ravenOptions, new IDataMigration[0]);
+                await service.StartAsync(CancellationToken.None).ConfigureAwait(false);
```

Comment:
What was the value in making this class a hosted service as opposed to just having an async method to do initialization/index creation?

Reply:
The hosted service already existed. All we are doing is calling into it. This is following the same pattern as the primary instance, which was already doing this. I like the idea of pulling the initialization/index creation out into a separate method. We don't really need to call stop because the document store is inside of a using statement.